docroot/themes/humsci/humsci_basic/src/scss/elements/_tables.scss

```diff
@@ -3,16 +3,8 @@
   overflow-x: auto;
 }
 
-table {
-  $border-color: lighten($su-color-driftwood, 15%);
-  $header-background-color: $su-color-warm-grey;
-  $header-foreground-color: $hb-color--white;
-  $header-border-color: $hb-color--white;
-
-  @include hb-colorful {
-    $header-background-color: darken(hb-colorful-variation(secondary), 10%);
-  }
-
+table,
+.hb-table-pattern {
```

Comment:
We should move these classes out of this file and into the  `_hb-table-pattern` sass partial and use mixins for these shared styles.